Doc/library/struct.rst

```diff
@@ -349,6 +363,11 @@ Notes:
    of bytes.  As a special case, ``'0s'`` means a single, empty string (while
    ``'0c'`` means 0 characters).
 
+(10)
+   For the ``'E'`` and ``'C'`` conversion codes, the packed representation uses
+   the IEEE 754 binary32 and binary64 format for components of the complex
```

Comment:
Again, link to the spec, we shouldn't expect people to have arbitrary specification numbers memorized.

Reply:
I doubt it's arbitrary.  We have many references to the IEEE 754 standard in docs.